cmd/eval-dev-quality/cmd/evaluate.go

```diff
@@ -182,8 +182,12 @@ func (command *Evaluate) Execute(args []string) (err error) {
 		}
 	}
 
+	isPlain := len(commandRepositories) == 0 || (len(commandRepositories) == 1 && commandRepositories["plain"])
```

Comment:
isPlain -> isPlainRepository

Reply:
Wait. what is the logic here? Why do we not ccategorize every repository? I know that we currently only have "plain" Go repository, but why do we already block new ones here?

Reply:
(don't think that this is necessary)

Reply:
cause for others we don't know the total score... will introduce a category unknown in that case